Makefile

```diff
@@ -239,13 +239,22 @@ install-travis:
 # Example:
 #   make build-rpms
 build-rpms:
-	OS_ONLY_BUILD_PLATFORMS='linux/amd64' tito build --test --rpm --no-cleanup --rpmbuild-options='--define "make_redistributable 0"'
+	OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpm-release.sh
 .PHONY: build-rpms
 
 # Build RPMs for all architectures
 #
 # Example:
 #   make build-rpms-redistributable
 build-rpms-redistributable:
-	tito build --test --rpm --no-cleanup --rpmbuild-options='--define "make_redistributable 1"'
+	hack/build-rpm-release.sh
 .PHONY: build-rpms-redistributable
+
+# Build a release of OpenShift using tito for linux/amd64 and the images that depend on it.
+#
+# Example:
+#   make release
+rpm-release: clean build-rpms
```

Comment:
So what's your take on this? You'll get RPMs and I imagine you could email them to someone ... `release-rpms`?